src/com/hms/flexyosisoftconnector/PayloadBuilder.java

```diff
@@ -260,6 +268,71 @@ public static String endOMFDataMessage() {
     return payload;
   }
 
+  /**
+   * Setup the format for how string data from this device will be stored with omf
+   *
+   * @return returns the JSON payload to construct a new Type message.
+   */
+  public static String getStringTypeBody() {
+
```

Comment:
Why is there an extra line in each method?

Reply:
blank line

Reply:
I like my extra lines. Give the line before it personal space. The other methods in here seem to not need personal space for that first line tho so I'll conform to the style of no space